custom_components/huesensor/binary_sensor.py

```diff
@@ -34,6 +34,10 @@
         "reachable",
         "sensitivity",
         "threshold",
+        "threshold_offset"
+    ],
+    "RWL": ["last_updated", "battery", "on", "reachable"],
```

Comment:
Isnt this related to remotes?

Reply:
yo're referring to line 39? yes, but it is already in the remote.py file. Should it stay in the binary_sensor.py too? Dont think the threshold_offset is for the remotes too, isn't it?

my files work locally as expected without this.

Reply:
Yes please remove from binary_sensor file